Makefile

```diff
@@ -0,0 +1,52 @@
+.DEFAULT_GOAL := all
+
+
+.PHONY: all
+all: ## Show the available make targets.
+	@echo "Usage: make <target>"
+	@echo ""
+	@echo "Targets:"
+	@fgrep "##" Makefile | fgrep -v fgrep
+
+.PHONY: clean
+clean: ## Clean the temporary files.
+	rm -rf .pytest_cache
+	rm -rf .mypy_cache
+	rm -rf .coverage
+	rm -rf .ruff_cache
+	rm -rf megalinter-reports
+
+.PHONY: format
+format:  ## Format the code.
+	poetry run black .
+	poetry run ruff check . --fix
+
+.PHONY: black
```

Comment:
Would these formatters also want to run on the `aws_lambda_scripts` directory, not just `src`?

Reply:
I'm not sure if it's necessary because it seems to just be loading in environment data pertaining to AWS, but i'll add it in anyways for good measure